src/main/java/com/yahoo/bullet/rest/service/StatusService.java

```diff
@@ -62,11 +64,8 @@ boolean hasResult() {
         }
     }
 
-    static final Query TICK_QUERY = new Query();
-    static {
-        TICK_QUERY.setAggregation(new Aggregation(1, Aggregation.Type.RAW));
-        TICK_QUERY.setDuration(1L);
-    }
+    static final Query TICK_QUERY = new Query(new Projection(), null, new Raw(1), null, new Window(), 1L);
+    static final String TICK_STRING = "Webservice status tick";
```

Comment:
Not sure if you saw this. Are you ok with this string (logged in backend)? @akshaisarma 
Would "Bullet service status tick" or something else look better? 🤔

Reply:
I thought this was better since what the query is doesn't really matter. But Bullet service status tick is also fine with me.